uint8       NofClusterNear
uint8       NofClusterFar
uint16      MeasCounter
uint8       InterfaceVersion
